packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.test.tsx

```diff
@@ -29,12 +29,11 @@ jest.mock("./util", () => ({
         mockedJoinLabelsAsSpokenMathForTests(input),
 }));
 
-const exampleEquationsMock = {
-    foo: ["bar", "zot"],
-};
 jest.mock("./locked-function-examples", () => ({
     __esModule: true,
-    default: exampleEquationsMock,
+    default: {
+        foo: ["bar", "zot"],
+    },
 }));
```

Comment:
Both babel and swc hoist `jest.mock()` calls to ensure they work correctly. Apparently with SWC though, the hoisting is stricter. This was complaining that it couldn't find the variable `exampleEquationsMock`. Because this was the only usage, I simply inlined it.